src/main/java/com/twilio/oai/TwilioJavaGenerator.java

```diff
@@ -183,9 +186,19 @@ public String toApiFilename(final String name) {
                         .filter(Objects::nonNull)
                         .map(this::getModel)
                         .flatMap(Optional::stream)
-                        .forEach(model -> resource.put("responseModel", model));
+                        .forEach(model -> {
+                            resource.put("responseModel", model);
```

Comment:
Should stick this line (with the above if condition) into the new `forEach` you've added below.

Reply:
Done